ext/intl/tests/bug76093.phpt

```diff
@@ -0,0 +1,81 @@
+--TEST--
+Bug #76093 (NumberFormatter::format loses precision)
+--SKIPIF--
+<?php if( !extension_loaded( 'intl' ) ) print 'skip'; ?>
```

Comment:
Please use the EXTENSIONS section, if other tests in ext/intl don't use it they need to be converted and backported (as a separate PR).

Reply:
Done.